package.json

```diff
@@ -268,6 +268,7 @@
     ]
   },
   "dependencies": {
+    "framer-motion": "^11.11.17",
```

Comment:
Was there a reason we added it to the root `package.json` rather than just updating it as part of our `ibm-products` package?

Reply:
No that was an accident!